### PR TITLE
Introduce confirmationEmail

### PR DESCRIPTION
**Pull request by JustCaptcha**
Testing:
1. yarn
2. yarn start:email

image
image
image
image

### CODE REVIEW

File: universal-login-relayer/package.json [hunk 1]
```diff
@@ -20,6 +21,8 @@
     "express-useragent": "^1.0.12",
     "geoip-lite": "^1.3.5",
     "knex": "^0.20.0",
+    "mjml": "^4.6.3",
+    "mjml-react": "^1.0.57",
```
Comment (sz-piotr) [inline universal-login-relayer/package.json:25]: Add react and react-dom

File: universal-login-relayer/package.json [hunk 2]
```diff
@@ -50,6 +54,7 @@
     "@types/express": "^4.16.1",
     "@types/express-useragent": "^0.2.21",
     "@types/geoip-lite": "^1.1.29",
+    "@types/mjml": "^4.0.4",
```
Comment (sz-piotr) [inline universal-login-relayer/package.json:57]: dependencies

File: universal-login-relayer/src/emailPlaygroundServer.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+import {confirmationEmailHtml} from './core/utils/confirmationEmailHtml';
+
+const port = 3000;
+const app = express();
+
+app.get('/', (req, res) => {
+  const confirmation = confirmationEmailHtml("uni-login-291862", '/somelink')
```
Comment (xitronix) [inline universal-login-relayer/src/emailPlaygroundServer.ts:8]: Inline confirmation
``` res.send(confirmationEmailHtml("uni-login-291862", '/somelink'));```

File: universal-login-relayer/src/core/utils/confirmationEmailHtml.tsx [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+  MjmlGroup,
+  MjmlWrapper
+} from 'mjml-react';
+
+interface ConfirmationEmailProps {
+  clipboardURL?: string;
+  code: string;
+}
+
+export const confirmationEmail = (code = 'uni-login-291862', clipboardURL = '') => {
```
Comment (xitronix) [inline universal-login-relayer/src/core/utils/confirmationEmailHtml.tsx:30]: In my opinion, we shouldn't have here default values

File: universal-login-relayer/src/core/utils/confirmationEmailHtml.tsx [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+  MjmlFont,
+  MjmlHero,
+  MjmlSpacer,
+  MjmlAttributes,
+  MjmlGroup,
+  MjmlWrapper
+} from 'mjml-react';
+
+interface ConfirmationEmailProps {
+  clipboardURL?: string;
```
Comment (xitronix) [inline universal-login-relayer/src/core/utils/confirmationEmailHtml.tsx:26]: Why clipboardURL is optional?

**Pull request merged**